src/mixins/createTabbedMixin.ts

```diff
@@ -248,18 +245,26 @@ const createTabbedMixin: TabbedMixinFactory = createRenderMixin
 					if (tabVNode.properties) {
 						(tabVNode as any).properties['data-visible'] = String(isActiveTab);
 					}
-					childrenNodes[idx] = tabVNode;
+					const childNode = {
```

Comment:
Ideally we would be explicit what the object literal type is.  It should be `DNode`, right?